packages/babel-plugin/__tests__/stylex-transform-create-test.js

```diff
@@ -1402,15 +1406,17 @@ describe('@stylexjs/babel-plugin', () => {
         "import _inject from "@stylexjs/stylex/lib/stylex-inject";
         var _inject2 = _inject;
         import stylex from 'stylex';
-        _inject2(".x1gykpug:hover{background-color:red}", 3130);
-        _inject2(".x11bf1mc:hover{color:var(--1ijzsae,revert)}", 3130);
+        _inject2(".xrkmrrc{background-color:red}", 3000);
+        _inject2(".x9lz66z{color:var(--4xs81a,revert)}", 3000);
+        _inject2("@media (min-width: 1000px){.xtljkjt.xtljkjt:hover{color:green}}", 3330);
+        _inject2(".x17z2mba:hover{color:blue}", 3130);
         export const styles = {
           default: color => [{
-            ":hover_backgroundColor": "x1gykpug",
-            ":hover_color": "x11bf1mc",
+            backgroundColor: "xrkmrrc",
+            color: "x9lz66z xtljkjt x17z2mba",
```

Comment:
This is what we actually want in this case:

```suggestion
            color: color == null ? "" : "x9lz66z " + "xtljkjt x17z2mba",
```